templates/success.html

```diff
@@ -12,11 +12,44 @@
 -->
 
 {% block page %}
-TODO
+<div class="shadow alert alert-success text-center mx-auto mt-4" style="width: 80%" role="alert">
+    <h1 class="alert-heading font-weight-bold" style="color: rgb(26, 118, 0);">提交成功</h1>
+    <i class="bi bi-check-circle min-vw-100 min-vh-100" style="font-size: 5rem; color: rgb(26, 118, 0);"></i>
+    <hr>
+    <p style="font-size: 1.5rem;">您填写的问卷 {{qname}} {{qid}} 已经成功提交</p>
+    <br>
+    <small class="text-muted">{{message}}</small>
+    <br>
+    <h6 id="lab" class="text-muted"></h6>
+    <p></p>
```

Comment:
```suggestion
<div class="mb-4" style="font-size: 1.5rem;">您填写的问卷 {{qname}} (#{{qid}}) 已经成功提交</div>
    <div class="text-muted">{{message}}</div>
    <div class="text-muted mb-2" id="lab"></div>
```
建议用 [margin](https://developer.mozilla.org/zh-CN/docs/Web/CSS/margin) 控制[块级元素](https://developer.mozilla.org/zh-CN/docs/Glossary/Block-level_content)的间距